pep-8001.rst

```diff
@@ -75,8 +75,14 @@ intend to remain inactive* to abstain from voting.
 When is the vote?
 -----------------
 
-The vote will happen in a 2-week-long window from November 16 2018
-to November 30 (Anywhere-on-Earth).
+November 16th, 2018 to November 30th, 2018 is the official governance
+PEP review period.  We discourage the PEP authors from making major
+substantive changes during this period, although it is expected that
+minor tweaks may occur, as the result of this discussion period.
+
+The vote will happen in a 2-week-long window from December 1st, 2018
+to December 16th, 2018
```

Comment:
Would it be possible to only open the vote 1 week, to only postpone the vote result by 1 week?